pkg/morph/client/reputation/put.go

```diff
@@ -0,0 +1,48 @@
+package reputation
+
+import (
+	"github.com/pkg/errors"
+)
+
+// PutArgs groups the arguments of "put reputation value" invocation call.
+type PutArgs struct {
+	epoch  uint64
+	peerID []byte
+	value  []byte
+}
+
+// SetEpoch sets epoch of reputation value.
+func (p *PutArgs) SetEpoch(v uint64) {
+	p.epoch = v
+}
+
+// SetPeerID sets peer id of reputation value.
+func (p *PutArgs) SetPeerID(v []byte) {
+	p.peerID = v
+}
+
+// SetValue sets marshaled reputation value.
+func (p *PutArgs) SetValue(v []byte) {
+	p.value = v
+}
+
+// Put invokes direct call of "put reputation value" method of audit contract.
+func (c *Client) Put(args PutArgs) error {
+	return errors.Wrapf(c.client.Invoke(
+		c.putMethod,
+		int64(args.epoch),
+		args.peerID,
+		args.value,
+	), "could not invoke method (%s)", c.putMethod)
+}
+
+// PutViaNotary invokes notary call of "put reputation value" method of
+// audit contract.
+func (c *Client) PutViaNotary(args PutArgs) error {
```

Comment:
Cosmetic: add boolean `notary` to `PutArgs` and leave only method `Put` (the same in `wrapper` pkg).

Reply:
I would like to keep two separate functions. Notary call is an add-on for client and it should not affect functions that can be used without notary, I think.